src/routes/docs/tutorials/sveltekit-ssr-auth/step-3/+page.markdoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: tutorial
+title: Initialize SDK
+description: Add Authentication to a SvelteKit project using Appwrite.
+step: 3
+---
+Before you can use Appwrite, you need to create the Appwrite `Client` and set the project ID and endpoint. 
+The client is then used to create services like `Databases` and `Account`, so they all point to the same Appwrite project.
+
+Create a function to build services you need in a file like `src/lib/server/appwrite.js` and **exporting the instances**.
+
+As part of the function, set the current user's session if they are logged in. This is done by accessing the session cookie from the request and calling the `setSession(session)` with the cookie value.
+
```

Comment:
Add a info tag to talk about why we create a new client every time.

Reply:
Done